components/DaemonConsole.qml

```diff
@@ -213,4 +213,4 @@ Window {
         color: "#2F2F2F"
         z: 2
     }
-}
\ No newline at end of file
+}
```

Comment:
Not my fault. I didn't change that. Not sure if is my text editor (I just tried with vim, nano, pluma) or git itself. Anyways it seems that my only option is to run windows(lol) to do this PR, because a text file that doesn't end in a newline isn't a textfile on unix :P  
See **https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file**

Reply:
I'm generally against indentation changes, but this change of making the file end with newline seems acceptable. It's small anyway and only a one time thing, while reducing annoyance for others going forward.